docs/building_blocks/vector_search/readme.md

```diff
@@ -1,42 +1,157 @@
-<!-- TODO: Replace this text with a summary of article for SEO -->
+<!-- What is vector search, and how can vector databases help? -->
 
 # Vector Search
 
-<!-- TODO: Cover image: 
-1. You can create your own cover image and put it in the correct asset directory,
-2. or you can give an explanation on how it should be and we will help you create one. Please tag arunesh@superlinked.com or @AruneshSingh (GitHub) in this case. -->
+![](../../assets/building_blocks/vector_search/vector-search-cover-sketch.png)
+
+## The exponential growth of data
```

Comment:
You cover the unstructured data aspect very well, but you focus on it too much - I'd like a more balanced view, because the vectorization of structured data is equally as important.

Couple examples:
1. Node2Vec and GraphSAGE help us embedd graph data / entity relationship structures, which means that the promiximity of the resulting vectors represents closeness in the original graph.
2. https://huggingface.co/blog/autoformer and similar are transformer-based models for timeseries prediction and they perform at SOTA level - which means that the embeddings you can extract from them are / should be a powerful way to describe and compare timeseries, for example for annomaly detection.
3. There is a bunch of work on tabular data embedding, for example https://research.yandex.com/blog/embeddings-for-numerical-features-in-tabular-deep-learning but I'm sure one could find more

Reply:
Makes sense! Will look into use cases that are structured.